app/services/model/AutoCompleteItem.scala

```diff
@@ -29,4 +29,8 @@ object AutoCompleteItem {
 
   def fromDocumentType(documents: List[DocumentType]): List[AutoCompleteItem] =
     documents map (d => AutoCompleteItem(s"${d.description} - ${d.code}", d.code))
+
+  def fromNationalAdditionalCode(codes: List[NationalAdditionalCode]): List[AutoCompleteItem] = {
+    codes map (c => AutoCompleteItem(c.value, c.value))
+  }
```

Comment:
Only style comment, but unnecessary parenthesis. It'll be good to keep convention like above for similar methods.

Reply:
Fixed